src/test/java/io/jenkins/plugins/analysis/warnings/recorder/DashboardPortletITest.java

```diff
@@ -0,0 +1,204 @@
+                + "&stapler-class=hudson.views.LastFailureColumn&%24class=hudson.views.LastFailureColumn"
+                + "&stapler-class=hudson.views.LastDurationColumn&%24class=hudson.views.LastDurationColumn"
+                + "&stapler-class=hudson.views.BuildButtonColumn&%24class=hudson.views.BuildButtonColumn"
+                + "&_.name=%23+Issues"
+                + "&_.id=&stapler-class=io.jenkins.plugins.analysis.core.columns.IssuesTotalColumn"
+                + "&%24class=io.jenkins.plugins.analysis.core.columns.IssuesTotalColumn"
+                + "&leftPortletWidth=50%25&rightPortletWidth=50%25"
+                + "&_.name=Static+analysis+issues+per+tool+and+job"
+                + "&_.id="
+                + "&stapler-class=io.jenkins.plugins.analysis.core.portlets.IssuesTablePortlet"
+                + "&%24class=io.jenkins.plugins.analysis.core.portlets.IssuesTablePortlet&core%3Aapply="
+                + "&Jenkins-Crumb=test"
+                + "&json=%7B%22name%22%3A+%22Test%22%2C+%22description%22%3A+%22%22%2C+%22filterQueue%22%3A+false%2C+"
+                + "%22filterExecutors%22%3A+false%2C+%22statusFilter%22%3A+%22%22%2C+%22recurse%22%3A+false%2C+%22test0"
+                + "%22%3A+%22true%22%2C+%22columns%22%3A+%5B%7B%22stapler-class%22%3A+%22hudson.views.StatusColumn"
+                + "%22%2C+%22%24class%22%3A+%22hudson.views.StatusColumn%22%7D%2C+%7B%22stapler-class%22%3A+"
+                + "%22hudson.views.WeatherColumn%22%2C+%22%24class%22%3A+%22hudson.views.WeatherColumn%22%7D%2C+%7B"
+                + "%22stapler-class%22%3A+%22hudson.views.JobColumn%22%2C+%22%24class%22%3A+%22hudson.views.JobColumn"
+                + "%22%7D%2C+%7B%22stapler-class%22%3A+%22hudson.views.LastSuccessColumn%22%2C+%22%24class%22%3A+"
+                + "%22hudson.views.LastSuccessColumn%22%7D%2C+%7B%22stapler-class%22%3A+%22hudson.views.LastFailureColumn"
+                + "%22%2C+%22%24class%22%3A+%22hudson.views.LastFailureColumn%22%7D%2C+%7B%22stapler-class"
+                + "%22%3A+%22hudson.views.LastDurationColumn%22%2C+%22%24class%22%3A+%22hudson.views.LastDurationColumn"
+                + "%22%7D%2C+%7B%22stapler-class%22%3A+%22hudson.views.BuildButtonColumn%22%2C+%22%24class%22%3A+"
+                + "%22hudson.views.BuildButtonColumn%22%7D%2C+%7B%22name%22%3A+%22%23+Issues%22%2C+%22selectTools"
+                + "%22%3A+false%2C+%22tools%22%3A+%7B%22id%22%3A+%22%22%7D%2C+%22stapler-class%22%3A+"
+                + "%22io.jenkins.plugins.analysis.core.columns.IssuesTotalColumn%22%2C+%22%24class%22%3A+"
+                + "%22io.jenkins.plugins.analysis.core.columns.IssuesTotalColumn%22%7D%5D%2C+%22includeStdJobList"
+                + "%22%3A+false%2C+%22topPortlet%22%3A+%7B%22name%22%3A+%22Static+analysis+issues+per+tool+and+job"
+                + "%22%2C+%22hideCleanJobs%22%3A+false%2C+%22showIcons%22%3A+false%2C+%22selectTools%22%3A+false%2C+"
```

Comment:
Any chance to extract the `false` for `hideCleanJobs` as parameter so we have a chance to test the hiding of clean jobs?